README.md

```diff
@@ -122,7 +124,7 @@ Options:
       --userAgentSuffix                     Append suffix to existing browser
                                             user-agent (ex: +MyCrawler,
                                             info@example.com)           [string]
-      --useSitemap                          If enabled, check for sitemaps at
+      --useSitemap, --sitemap               If enabled, check for sitemaps at
```

Comment:
do we want to aslo allow usesitemap since we do that for some other flags

Reply:
Eh, maybe better to avoid so many variations?